### PR TITLE
https #121

### PR DESCRIPTION
**Pull request by balthazzar**
https://github.com/webRunes/Chess-WRIO-Game/issues/121

change tabs to spaces

### CODE REVIEW

File: src/server/appData/utils.js [hunk 1]
```diff
@@ -1,75 +1,75 @@
...
+            return console.log(err.message);
+        } else if (result.result.n === 0) {
+            webRunes_AppData.insert({
+                appName: APP_NAME,
+                data: data
+            }, function(err, res) {
+                if (err) {
+                    return console.log(err.message);
+                } else {
+                    return console.log('App data:', 'Application data have been sent to database');
```
Comment (AlexeyAnshakov) [inline src/server/appData/utils.js]: Application data has been sent

File: src/server/appData/utils.js [hunk 2]
```diff
@@ -1,75 +1,75 @@
...
+                }
+            });
+        } else {
+            return console.log('App data:', 'Application data have been sent to database');
```
Comment (AlexeyAnshakov) [inline src/server/appData/utils.js]: Application data has been sent

File: src/server/appData/utils.js [hunk 3]
```diff
@@ -1,75 +1,75 @@
...
-					resolve();
-				} else {
-					console.log('App data:', 'Application data have been load from database')
-					resolve(results[0].data);
-				}
+                if (results.length === 0) {
+                    console.log('App data is not found');
+                    resolve();
+                } else {
+                    console.log('App data:', 'Application data have been load from database')
```
Comment (AlexeyAnshakov) [inline src/server/appData/utils.js]: Application data loaded

File: src/server/appData/utils.js [hunk 4]
```diff
@@ -1,75 +1,75 @@
...
 
-				if (results.length === 0) {
-					console.log('App data is not found');
-					resolve();
-				} else {
-					console.log('App data:', 'Application data have been load from database')
-					resolve(results[0].data);
-				}
+                if (results.length === 0) {
+                    console.log('App data is not found');
```
Comment (AlexeyAnshakov) [inline src/server/appData/utils.js]: App data not found

File: src/server/utils/access.js [hunk 5]
```diff
@@ -1,206 +1,206 @@
...
+                titterID: titterID
+            })
+            .toArray(function(err, data) {
+                if (data && data[0]) {
+                    twitter.verifyCredentials(data[0].token, data[0].tokenSecret, function(error, _data, res) {
+                        resolve(error)
+                    });
+                } else {
+                    reject({
+                        message: "no user"
```
Comment (AlexeyAnshakov) [inline src/server/utils/access.js]: User not found

File: src/server/wrio_nconf.js [hunk 6]
```diff
@@ -3,14 +3,14 @@ var path = require('path');
 var fs = require('fs');
 
 nconf.env()
-	.argv();
+    .argv();
 nconf.file(path.resolve(__dirname, '../../config.json'));
 
 var defaultUrl = 'http://titter' + nconf.get('db:workdomain');
```
Comment (AlexeyAnshakov) [inline src/server/wrio_nconf.js]: https

File: src/server/wriologin.js [hunk 7]
```diff
@@ -3,83 +3,83 @@
...
-		}, function(err, session) {
-			if (err || !session) {
-				console.log("loginWithSessionId:User not found", err);
-				done(err || !0);
-				return;
-			}
+    function loginWithSessionId(ssid, done) {
+        var match = ssid.match(/^[-A-Za-z0-9+/=_]+$/m);
+        if (!match) {
+            console.log("Wrong ssid");
```
Comment (AlexeyAnshakov) [inline src/server/wriologin.js]: ssid invalid

File: src/server/wriologin.js [hunk 8]
```diff
@@ -3,83 +3,83 @@
...
+            done("Error");
+            return
+        }
+        console.log("Trying deserialize session", ssid);
```
Comment (AlexeyAnshakov) [inline src/server/wriologin.js]: Deserialize session attempt

File: src/server/wriologin.js [hunk 9]
```diff
@@ -3,83 +3,83 @@
...
 
-			if (user != undefined) {
-				deserialize(user, done);
-			} else {
-				done("Wrong cookie")
-			}
+            if (user != undefined) {
+                deserialize(user, done);
+            } else {
+                done("Wrong cookie")
```
Comment (AlexeyAnshakov) [inline src/server/wriologin.js]: Cookie invalid

File: src/server/wriologin.js [hunk 10]
```diff
@@ -3,83 +3,83 @@
...
+                console.log("Error executing request");
+                done(err);
+            } else {
+                if (res.token && res.tokenSecret) {
+                    done(null, {
+                        "token": res.token,
+                        "tokenSecret": res.tokenSecret
+                    })
+                } else {
+                    done("No login with twitter");
```
Comment (AlexeyAnshakov) [inline src/server/wriologin.js]: Can't login with twitter

**Comment by balthazzar** [comment]
@AlexeyAnshakov fixed

**Pull request merged**